scene/main/viewport.h

```diff
@@ -401,6 +401,7 @@ class Viewport : public Node {
 	DefaultCanvasItemTextureRepeat default_canvas_item_texture_repeat = DEFAULT_CANVAS_ITEM_TEXTURE_REPEAT_DISABLED;
 
 	bool disable_input = false;
+	bool disable_input_override = false;
```

Comment:
Since this is only required for "Game Editor", all related code should be wrapped in TOOLS_ENABLED, no?
Although camera overrides weren't wrapped, so we can decide on that later and wrap everything.